src/helpers/createTimeoutController.ts

```diff
@@ -1,6 +1,11 @@
 import { WaitOptions } from '../types'
+import { jestFakeTimersAreEnabled } from './jestFakeTimersAreEnabled'
+const DEFAULT_TIMEOUT = 1000
 
-function createTimeoutController(timeout: WaitOptions['timeout']) {
+function createTimeoutController(
+  timeout: WaitOptions['timeout'] = DEFAULT_TIMEOUT,
```

Comment:
@mpeyper This line is little confusing. Since some time it is not timeout but interval 
We may need to reword it and give WaitOptions['timeout'] |WaitOptions['interval'] type for it.

Reply:
It's probably unlikely the types for `timeout` and `interval` will ever diverge, but I see your point.

Perhaps were better off typing it explicitly as what it allows (i.e. `number | false`) and because this function is actually not related to `WaitOptions` other than by coincidence.

Reply:
I changed the type to number | false. I think it is ready to merge now